README.zh-CN.md

```diff
@@ -21,23 +21,30 @@
 
 # [Apache APISIX](https://github.com/apache/apisix) 控制台
 
+Apache APISIX Dashboard 项目的目标是为了让大家快速上手体验和学习 Apache APISIX，并不能直接用于生产环境。所以它的功能点覆盖永远都是 Apache APISIX 的子集，也可能会滞后于 Apache APISIX 的快速迭代。如果你需要把 Dashboard 项目用于生产系统，需要对用户权限、通讯安全、高可用、高级功能等方面做增强。
+
 ## 用户指南
 
 请参考 [用户指南](./docs/USER_GUIDE.zh-CN.md)
 
-## 部署
+## 开发者
+
+如果你是开发者，请参考这里分别启动 manager-api 和 web
+
+- [依赖检查](#依赖检查)
+- [开发 Dashboard](./docs/develop.zh-CN.md)
+
+## 部署运行
 
-当前支持如下方式部署：
+我们有多种部署方式
 
-- [手动部署](./docs/deploy.zh-CN.md)
-- [使用 Docker 部署](./docs/deploy-with-docker.zh-CN.md)
+- [使用 Docker 一键部署](./docs/deploy-with-docker.zh-CN.md)
 
-## 开发
+- [从源文件打包部署](./docs/deploy.zh-CN.md)
```

Comment:
This should be the first than Deploy with Docker

Reply:
and Build, Deploy, and Run is not the same at all.

Reply:
the order ref to `prometheus`